R/bundle.R

```diff
@@ -0,0 +1,133 @@
+##' Create a bundle of tasks.  This is simply a collection of tasks
+##' that relate together in some way, and we provide some helper
+##' functions for working with them that save you writing lots of
+##' loops.  Each bundle has a name, which will be randomly generated
+##' if you don't provide one, and a set of task ids.
+##'
+##' @title Create task bundle
+##'
+##' @param ids A character vector of task ids
+##'
+##' @param validate Boolean, indicating if we should check that the
```

Comment:
I think elsewhere we say Logical - couple of exceptions on task_eval:19 and windows.R:31

Reply:
sure, I'll make that consistent

Reply:
updated throughout now - I think the idea was that as string was a scalar character, then boolean might be a scalar logical? but that is probably a relationship only in my head